conclave-enclave/src/main/kotlin/com/r3/conclave/enclave/Enclave.kt

```diff
@@ -244,6 +247,24 @@ abstract class Enclave {
         this.env = env
         initCryptography()
 
+        // Prevent users from printing to the console when running the enclave in release mode
+        // GraalVM already discards the output so the following if statement is only relevant for GramineSGX
+        // N.B. The user can still override the configuration below and print to the console while running
+        // the enclave in release mode. But that has to be a conscious decision.
```

Comment:
I'm not sure the final statement is correct. How can the user enclave re-apply the stdout again? I don't think they can, but let me how they could.

Reply:
The last sentence might not be clear. By user, I am referring to the developer of the enclave. Please let me know if that is the case and I will update the last sentence to become more explicit.

The code to redirect the output again to the stdout is `System.setOut(PrintStream(FileOutputStream(FileDescriptor.out)))`.